tutorials/vue/vue-file-upload.md

```diff
@@ -0,0 +1,9 @@
+Using Vue 2 with Axios, you can upload files easily with a few clicks.
+Using the `<input>` tag and specifying the type to file, the browser will
+allow you to select the file to upload from your computer. You will need
+to create a form and so we recommend the form-data module to use for
```

Comment:
You only need the form-data module for Node, `FormData` is a built in class in browsers.